example/deep-embedded-clustering/dec.py

```diff
@@ -82,8 +82,8 @@ def list_arguments(self):
 
     def setup(self, X, num_centers, alpha, save_to='dec_model'):
         sep = X.shape[0]*9/10
-        X_train = X[:sep]
-        X_val = X[sep:]
+        X_train = X[:int(sep)]
```

Comment:
calc sep with // instead

Reply:
I made the change here, testing locally now.